interfaces/IBF-dashboard/src/app/services/map.service.ts

```diff
@@ -1008,7 +1003,7 @@ export class MapService {
     if (
       !area.triggerValue ||
       area.triggerValue < 1 ||
-      !this.eventState?.event?.thresholdReached
```

Comment:
@arsforza are you sure this is OK. I feel this stuff happened to me also, which was because of the '@typescript-eslint/no-unnecessary-condition': 'warn' rule, which I switched to 'off' in my PR, because it was actually removing question marks which lead to breaking changes.